relayer/storage/redis-storage.ts

```diff
@@ -104,6 +104,8 @@ export class RedisStorage implements Storage {
             this.opts.redisCluster,
           )
         : new Redis(this.opts.redis);
+
+    // TODO: consider using a queue per chain
```

Comment:
@gzimmermann @jhowarth: I'm inclined to think that we should split the bullmq queue by chain.

The reason for this is that currently we have to configure concurrency on two places:
- wallets-concurrency (it's a manual setup right now, but the concurrency we can handle is ultimately the same as the count of wallets the relayer has a for a particular chain)
- bull-mq concurrency: which ultimately represents the amount of max active workflows at any given time

A practical approach to configure this would be to set the concurrency of bull-mq = `supported-chains * concurrency each chain can handle`, which is in practice the maximal concurrency we might ever handle.

This can generate some weird conditions, for example:
make some queues processing unnecesarily efficiently: we might have cases in which all the active workflows (bull-mq concurrency) are for the same chain, and so we might have VAAs in the queue waiting to be processed while the wallet for that chain isn't processing anything

Do you have strong feelings about this? any reason to think that this is not a good idea? Is it too risky to do it in this particular moment?

Reply:
Joe stated on a conversation with me that given that VAAs are a few kb each, there's no harm in just setting the bull-mq concurrency really high. The higher we configure bullmq concurrency, the less likely it is that all jobs will be taken for a single chain. This doesn't scale, but doesn't require any effort either. For the moment we might consider this just a tech debt and move on, but I'd like to address it

Reply:
"I'm inclined to think that we should split the bullmq queue by chain"

FYI, this would require parsing the payload of the VAA prior to sending to a queue. This is a non trivial requirement. As long as you're dealing with token bridge vaas or cctp vaas I guess we could do that, but we won't always know what the chain is.